gems/aws-partitions/lib/aws-partitions/partition.rb

```diff
@@ -21,8 +21,8 @@ def region(region_name)
         if @regions.key?(region_name)
           @regions[region_name]
         else
-          msg = "invalid region name #{region_name.inspect}; valid region "
-          msg << "names include #{@regions.keys.join(', ')}"
+          msg = "invalid region name #{region_name.inspect}; valid region "\
+            "names include #{@regions.keys.join(', ')}"
```

Comment:
As per Rubocop guidance, i believe that these strings should be aligned such as:

```
msg = "stuff"\
      "other stuff"
```

Reply:
Working on it!